web/blog/tenzir-v4.9/index.md

```diff
@@ -46,10 +112,31 @@ export --live
 | enrich indicators --field dns.rrname
 ```
 
+The `enrich` operator gained a new `--filter` option to remove events it could
```

Comment:
Above this (can't directly comment on it because it's not part of this PR):
~~~
If you have massive amounts of indicators or a large amount of things you would
like>>>t<<< to contextualize, this feature is for you.
~~~